news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -280,16 +280,16 @@ protected function getMetaFields($objArticle)
 	{
 		$meta = StringUtil::deserialize($this->news_metaFields);
 
+		$return = array('date' => '', 'author' => '');
```

Comment:
I don't think that this is correct. What problem does it fix?

Reply:
This was suggested in an earlier review comment and I [pushed an additional commit](https://github.com/contao/contao/pull/2848/commits/bf5a1da42c610a9085fb9348759e8adfea9d073b) as a result. Previously, I was fixing the problem with `?? null` in lines 151 and 156. Do you prefer that?

Reply:
Yes.

Reply:
Done. I removed the review commit and cleaned up my commits a bit.